src/Catty/PlayerEngine/DataModel/CBSpriteNode/CBSpriteNode.swift

```diff
@@ -176,6 +176,15 @@ class CBSpriteNode: SKSpriteNode {
         spriteObject.lookList
     }
 
+    @objc func indexLook(_ index: Int) -> Look? {
```

Comment:
In case you outsource this method directly to the `CBSpriteNode`, please add a test for it (`CBSpriteNodeTests`) and rename to `look(for index: Int)`.

In that case please also use this logic for the `SetLookBrick`'s instruction.